packages/yourloops/test/integration/pages/notification/notification-for-patient.spec.tsx

```diff
@@ -0,0 +1,126 @@
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { act, within, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderPage } from '../../utils/render'
+import { mockPatientLogin } from '../../mock/patient-login.mock'
+import { monitoredPatientAsTeamMember } from '../../mock/patient.api.mock'
+import NotificationApi from '../../../../lib/notifications/notification.api'
+import { Notification } from '../../../../lib/notifications/models/notification.model'
+import { TeamMemberRole } from '../../../../lib/team/models/enums/team-member-role.enum'
+import { NotificationType } from '../../../../lib/notifications/models/enums/notification-type.enum'
+import { invitationHcpForPatient, teamOne, teamTwo } from '../../mock/team.api.mock'
+import TeamAPI from '../../../../lib/team/team.api'
+
+describe('Add patient in the team with a notification', () => {
+  const invitationTeam: Notification = {
+    id: '26a11710e98e4',
+    type: NotificationType.careTeamPatientInvitation,
+    metricsType: 'join_team',
+    email: 'ylp.ui.test.67-patient12@diabeloop.fr',
+    creatorId: 'dec0816009d32',
+    date: '',
+    target: { id: '63c7b7989cacc878ecce2c40', name: 'sysReq-67-team2' },
+    role: TeamMemberRole.patient,
+    creator: {
+      userid: 'dec0816009d32',
+      profile: {
+        fullName: 'Backloops test SysReq 67 hcp1',
+        patient: {
+          birthday: '',
+          diagnosisDate: ''
+        },
+        email: ''
+      }
+    }
+  }
+
+  beforeAll(() => {
+    mockPatientLogin(monitoredPatientAsTeamMember)
+    jest.spyOn(TeamAPI, 'getTeams').mockResolvedValue([teamOne, teamTwo])
+    jest.spyOn(NotificationApi, 'getReceivedInvitations').mockResolvedValue([invitationTeam])
+    jest.spyOn(TeamAPI, 'getTeamFromCode').mockResolvedValue(invitationHcpForPatient)
+    jest.spyOn(NotificationApi, 'acceptInvitation').mockResolvedValue()
+  })
+
+  it('Should cancel add team', async () => {
```

Comment:
So this test is checking that we are able to close the dialog without accepting the team invite? Can you merge it with the test below? It is better to have on big test (faster on the CI)